remote_vector_index_builder/core/common/models/index_builder/ivf_pq_build_cagra_config.py

```diff
@@ -0,0 +1,50 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+from dataclasses import dataclass
+
+
+@dataclass
+class IVFPQBuildCagraConfig:
+    # The number of inverted lists (clusters)
+    # Hint: the number of vectors per cluster (`n_rows/n_lists`) should be
+    # approximately 1,000 to 10,000.
+    n_lists: int = 1000
+
+    # The number of iterations searching for kmeans centers (index building).
+    kmeans_n_iters: int = 10
+    # The fraction of data to use during iterative kmeans building.
+    kmeans_trainset_fraction: float = 0.1
+
+    # The bit length of the vector element after compression by PQ.
+    # Possible values: [4, 5, 6, 7, 8].
+    # Hint: the smaller the 'pq_bits', the smaller the index size and the
+    # better the search performance, but the lower the recall.
+    pq_bits: int = 8
```

Comment:
for all these values please find out the defaults from Faiss Cagra and lets use those defaults.

Reply:
updated